examples/tutorial/tests/conftest.py

```diff
@@ -8,15 +8,16 @@
 from flaskr.db import init_db
 
 # read in SQL for populating test data
-with open(os.path.join(os.path.dirname(__file__), "data.sql"), "rb") as f:
-    _data_sql = f.read().decode("utf8")
+with open(Path(__file__).parent / "data.sql", encoding="utf8") as f:
```

Comment:
I guess this is ok? You want to use `Path.read_text()` here if you're looking to simplify things. This would make more sense as part of #3557.